networktable.c

```diff
@@ -39,23 +38,41 @@ bool create_network_table(void)
 	return true;
 }
 
-// Read kernel's ARP cache using procfs
-void parse_arp_cache(void)
+bool create_network_addresses_table(void)
```

Comment:
As part of this DB upgrade, all entries in the `network` table should have a row in `network_addresses` using the value of the `ip` column from the `network` table. Then this `ip` column can be removed from `network`.